src/main/java/org/dataconservancy/packaging/shared/IpmPackager.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright 2016 Johns Hopkins University
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dataconservancy.packaging.shared;
+
+import org.apache.jena.rdf.model.*;
+
+import org.dataconservancy.packaging.tool.api.Package;
+import org.dataconservancy.packaging.tool.api.PackageGenerationService;
+import org.dataconservancy.packaging.tool.impl.IpmRdfTransformService;
+import org.dataconservancy.packaging.tool.model.*;
+import org.dataconservancy.packaging.tool.model.ipm.Node;
+
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Properties;
+
+/**
+ * Creates a DC package from a content provider, metadata and generation parameters.
+ * Created by wbt on 12/01/16.
+ */
+public class IpmPackager {
+
+    // TODO - There is a GitHub issue in osf-packaging about this use of this system property.
+    private String PACKAGE_LOCATION = System.getProperty("java.io.tmpdir");
+    private String PACKAGE_NAME = "MyPackage";
```

Comment:
These had been static final in the other project, and I didn't change their name (I tried to change as little as possible).  I also like camel case for private fields and will change these.